jams.js

```diff
@@ -14,6 +14,8 @@ ANY          ::= (SAFE | WS | SYN | #x5C)
 SAFE         ::= #x21 | [#x23-#x5A] | [#x5E-#x7A] | #x7C | #x7E
 `)
 
+// Expect a well-formed JSON string, already decoded, not UTF-8/UTF-16 encoded bytes etc.
```

Comment:
This comment is not clear, implies the function is accepting, well, a JSON string, probably different interpretation than you meant

Reply:
You're right. I meant `jams` expects a JAMS string, but the leave string have to be JSON strings.